addons/account/models/account_move.py

```diff
@@ -1220,7 +1222,12 @@ def _compute_payment_state(self):
     @api.depends('payment_state', 'state')
     def _compute_status_in_payment(self):
         for move in self:
-            move.status_in_payment = move.state if move.state == 'cancel' else move.payment_state
```

Comment:
I think you need to add the is_move_sent in the depends, and the colors in the list view aren't correct for the posted state